### ISSUE TITLE
[A11Y]  Graphic elements should be labeled 

### ISSUE THREAD

**Issue by softvision-oana-arbuzov** [webcompat/webcompat.com#3456]
**Browser / Version**: Firefox Nightly 81.0a1 (2020-08-18)
**Operating System**: Windows 10 Pro

**Steps to Reproduce**:
1. Navigate to https://webcompat.com/issues/new
2. Fill in all required info until reaching "What is wrong with the page" section.
3. Observe categories images.

**Expected Behavior:**
Categories images have labels.

**Actual Behavior:**
Categories images have no label.

**Notes:**
1. Screenshot attached.
![image](https://user-images.githubusercontent.com/12184325/90652338-a4482100-e246-11ea-9293-859eba200c45.png)
2. The issue is reproducible for all graphic elements (except the actual images ) 
- categories
![image](https://user-images.githubusercontent.com/12184325/90653169-8929e100-e247-11ea-829d-ab3e794ca3f8.png)
- browsers
![image](https://user-images.githubusercontent.com/12184325/90652712-0bfe6c00-e247-11ea-9058-9a5a50e4367b.png)
- arrows icons
![image](https://user-images.githubusercontent.com/12184325/90652788-1de00f00-e247-11ea-9643-dc7d076a023a.png)
- report progress icons
![image](https://user-images.githubusercontent.com/12184325/90652848-2f291b80-e247-11ea-82b5-218c948cb4d9.png)

2. Read more about it here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Text_labels_and_names

**Watchers:**
@softvision-oana-arbuzov
@cipriansv

**Comment by softvision-oana-arbuzov** [comment]
Affected area:
```html
<div class="icon-container">
	<img src="/img/svg/icons/svg-problem-mobile-vs-desktop.svg">
</div> Desktop site instead of mobile site
```

Proposal for categories/browsers graphic:
Adding `alt` attribute to <img> fixes the issues.
 ```html
<img src="/img/svg/icons/svg-problem-mobile-vs-desktop.svg" alt="Desktop site instead of mobile site">
```

**Comment by miketaylr** [comment]
Thanks, we should fix this.

Looking at https://html.spec.whatwg.org/multipage/images.html#alt

>  the alt attribute's value should never contain text that could be considered the image's caption, title, or legend. It is supposed to contain replacement text that could be used by users instead of the image; it is not meant to supplement the image. 

and 

> Another corollary is that the alt attribute's value should not repeat information that is already provided in the prose next to the image.

Given these, I think we can fix this by adding an empty `alt=""` attribute -- it's a purely decorative image and the `<label>` already has text.

**Comment by softvision-oana-arbuzov** [comment]
This issue is partially fixed:
-[OK] categories
-[OK] browsers
-[Not OK] arrows icons 
![image](https://user-images.githubusercontent.com/12184325/93609675-cfa06600-f9d4-11ea-9159-8374673302e7.png)
-[Not OK] report progress icons  
![image](https://user-images.githubusercontent.com/12184325/93609715-d9c26480-f9d4-11ea-9a43-4cd131c49504.png)

**Tested with:**
Browser / Version: Firefox Nightly 82.0a1 (2020-09-17)
Operating System: Windows 10 Pro

Reopening for further investigation.

**Comment by karlcow** [comment]
@miketaylr are there more things that need to be done here?

**Comment by miketaylr** [comment]
It looks like we still need to add empty `alt=''` to the arrow icons. I can send in a PR!

**Comment by miketaylr** [comment]
> It looks like we still need to add empty `alt=''` to the arrow icons.

So it seems slightly more complicated. These images aren't actually `<img>` elements, but a div using the `content` property via CSS:

```html
<div class="circle">
        <div class="number">2</div>
        <div class="icon"></div>
</div>
```

with

```css
.step.complete .icon {
    content: url(dd8ea4f7ca33ade7ec066633757f1926.svg);
    display: none;
    height: auto;
    width: 8px;
}
```

It might be possible to address this via aria somehow. Will look.
